src/plugin/ipc/file/fileconnection.cpp

```diff
@@ -518,38 +518,6 @@ void FileConnection::doLocking()
   _ckpted_file = false;
 }
 
-void FileConnection::handleUnlinkedFile()
-{
-  if ((!jalib::Filesystem::FileExists(_path) && !_isBlacklistedFile(_path)) ||
-      _type == FILE_DELETED ||
-      Util::strStartsWith(jalib::Filesystem::BaseName(_path), ".nfs")) {
-    /* File not present in Filesystem.
-     * /proc/self/fd lists filename of unlink()ed files as:
-     *   "<original_file_name>(deleted)"
-     */
-    string currPath = jalib::Filesystem::GetDeviceName(_fds[0]);
-
-    if (Util::strEndsWith(currPath, DELETED_FILE_SUFFIX)) {
-      JTRACE("Deleted file") (_path);
-      _type = FILE_DELETED;
```

Comment:
If I understand this code, we're going to stop checking for a suffix string " (deleted)".  Or do we check for it somewhere else when it's a suffix?  Could you explain further why this code can be deleted?  Thanks.